lib/ovirt_metrics.rb

```diff
@@ -112,6 +112,8 @@ def self.metrics_to_hashes(metrics, related_metrics, column_definitions, href_me
       values = {}
 
       column_definitions.each do |evm_col, info|
+        next if column_definitions == VM_COLUMN_DEFINITIONS && options[:metric].vm_status.to_i == 0
```

Comment:
Can you make a column or perhaps a variable that explains that `.vm_status.to_i == 0` means that a vm is not running.  It is not obvious from the code.